### PR TITLE
adding swagger docs, starting to add an authenticated endpoint

### PR DESCRIPTION
**Pull request by spruce-bruce**
#30

### CODE REVIEW

File: api/bootstrap.js [hunk 1]
```diff
@@ -1,11 +1,19 @@
 const ioc = require('electrolyte');
-const config = require('./config');
+// const config = require('./config');
```
Comment (tdboone) [inline api/bootstrap.js]: Should this still be here, commented out like this?

File: api/package.json [hunk 2]
```diff
@@ -32,7 +33,7 @@
     "handlebars": "^4.0.5",
     "hapi": "^16.1.0",
     "hapi-auth-jwt2": "^7.2.4",
-    "hapi-email-kue": "^0.3.0",
+    "hapi-swagger": "^7.7.0",
```
Comment (tdboone) [inline api/package.json]: You don't need this package in here anymore, right?

File: api/bootstrap.js (hunk 1 above)
Comment (spruce-bruce) [inline api/bootstrap.js]: Next PR removes this

File: api/package.json (hunk 2 above)
Comment (spruce-bruce) [inline api/package.json]: RIIIIIIIIGHT

File: api/bootstrap.js (hunk 1 above)
Comment (tdboone) [inline api/bootstrap.js]: Yeah, I saw that

**Pull request merged**